test/io/input/pipeprocesstester.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+sum_to () (
+    set -- $(seq $1)
+    IFS=+
```

Comment:
I think it sums the numbers by redirecting the string created from numeric values separated by `+` to `bc`. So it calculates `1+2+3+4+5+..` by concatenating and sending this string to separate process to do the calculations :D